client/lib/plans/constants.js

```diff
@@ -424,6 +434,59 @@ const getPlanBusinessDetails = () => ( {
 	getHiddenFeatures: () => [ FEATURE_AUDIO_UPLOADS ],
 } );
 
+const getPlanEcommerceDetails = () => ( {
+	group: GROUP_WPCOM,
+	type: TYPE_ECOMMERCE,
+	getTitle: () => i18n.translate( 'eCommerce' ),
+	getAudience: () => i18n.translate( 'Best for eCommerce sites' ),
+	getBlogAudience: () => i18n.translate( 'Best for online stores' ),
+	getPortfolioAudience: () => i18n.translate( 'Best for online stores' ),
+	getStoreAudience: () => i18n.translate( 'The plan for stores and small businesses' ),
+	getDescription: () => {
+		return i18n.translate(
+			'{{strong}}Best for eCommerce:{{/strong}} Start your' +
+				' online store with a specifically designed WooCommerce software bundle, unlimited' +
+				' storage, premium themes, and the ability to remove WordPress.com branding.',
+			{
+				components: {
+					strong: (
+						<strong className="plans__features plan-features__targeted-description-heading" />
+					),
+				},
+			}
+		);
+	},
+	getTagline: () =>
+		i18n.translate(
+			'Learn more about everything included with eCommerce and take advantage of its powerful marketplace features.'
+		),
+	getPlanCompareFeatures: () =>
+		compact( [
+			// pay attention to ordering, shared features should align on /plan page
+			FEATURE_CUSTOM_DOMAIN,
+			FEATURE_JETPACK_ESSENTIAL,
+			FEATURE_EMAIL_LIVE_CHAT_SUPPORT,
+			FEATURE_UNLIMITED_PREMIUM_THEMES,
+			FEATURE_ADVANCED_DESIGN,
+			FEATURE_UNLIMITED_STORAGE,
+			FEATURE_NO_ADS,
+			isEnabled( 'automated-transfer' ) && FEATURE_UPLOAD_PLUGINS,
+			isEnabled( 'automated-transfer' ) && FEATURE_UPLOAD_THEMES,
```

Comment:
Shouldn't this be Atomic v2-specific flag, like `atomic`?

Reply:
Yes, this whole section should be reviewed because I tried to find features that kind of matched the ones that are defined in the bundle on WordPress.com side.